packages/instruments/README.md

```diff
@@ -0,0 +1,30 @@
+## `@envelop/instruments`
+
+This package contains utility functions and types to ease the use of instruments accross Envelop,
+Yoga, whatwg-node and Hive Gateway plugins.
+
+> [!NOTE] Instruments are automatically composed together. This should only be used if the default
+> ordering doesn't suite your need (ie. instruments and hooks should not be executed in the same
+> order)
+
+## `composeInstruments(instruments: Instruments[]): Instruments`
+
+This function composes all the instruments into one. The instruments will be called in the same
+order than they are in the array (first is outer most call, last is inner most).
+
+```ts
+import { getInstrumentsAndPlugins } from '@envelop/instruments'
```

Comment:
This import is not used. Did you mean:
```suggestion
import { composeInstruments } from '@envelop/instruments'
```

or

```suggestion
import { composeInstruments } from '@envelop/core'
```

if it's coming from core?

Reply:
Yeah it's a leftover, I thought it would be useful but in fact it's not.

It is coming from core because core re-exports `@envelop/instruments`. Is though it would be easier like this, since the core already depends on it.